tests/integration/targets/azure_rm_aksagentpool/tasks/main.yml

```diff
@@ -33,6 +33,42 @@
       outbound_type: loadBalancer
   register: output
 
+- name: Get managecluter admin credentials
```

Comment:
Fwiw here in many other place I guess you misspelled `cluster` as `cluter`.

Reply:
@em- Thanks for your review! Already modify!